index.js

```diff
@@ -9,7 +9,9 @@ const pluginDefaults = {
     'default-src': ['https:'],
     'report-uri': 'http://localhost/csp_reports'
   },
-  headerKey: 'Content-Security-Policy-Report-Only'
+  headerKey: 'Content-Security-Policy-Report-Only',
+  policyHeaderKey: 'Content-Security-Policy',
```

Comment:
lets make this dummy proof and have the option be `upgradeInsecureRequests: true`